Snakefile

```diff
@@ -322,24 +334,27 @@ if config["US"].get("retrieve_US_databundle", True):
 
     rule retrieve_pumas:
         params:
-            gdrive_url="https://drive.google.com/drive/folders/1sWDPC1EEzVtgixBb8C-OqZiEX3dmTOec",
+            #gdrive_url="https://drive.google.com/drive/folders/1sWDPC1EEzVtgixBb8C-OqZiEX3dmTOec",
+            gdrive_url="https://drive.google.com/drive/folders/1A8rA2p1a1cv1poPYlBRvAuLeYxENA-KK?usp=drive_link",
             cookies_path=pathlib.Path(".cache", "gdown"),
-            output_directory=pathlib.Path("analysis", "gdrive_data", "data"),
+            cookie_filename = "pumas",
+            output_directory=pathlib.Path("analysis", "gdrive_data", "data", "utilities","ipums_puma_2010"),
             delta_months=5,
+            merge_files=False,            
         output:
-            expand(
-                "analysis/gdrive_data/data/utilities/ipums_puma_2010/{filename}",
-                filename=[
-                    "ipums_puma_2010.CPG",
-                    "ipums_puma_2010.sbn",
-                    "ipums_puma_2010.shp.xml",
-                    "ipums_puma_2010.dbf",
-                    "ipums_puma_2010.sbx",
-                    "ipums_puma_2010.shx",
-                    "ipums_puma_2010.prj",
-                    "ipums_puma_2010.shp",
-                ],
-            ),
+#            expand(
+#                "analysis/gdrive_data/data/utilities/ipums_puma_2010/{filename}",
+#                filename=[
+#                    "ipums_puma_2010.CPG",
+#                    "ipums_puma_2010.sbn",
```

Comment:
I guess the idea is to remove the commented-out fragment in the next iterations?